.github/workflows/qa-testbench.yaml

```diff
@@ -78,35 +71,56 @@ jobs:
           roleId: ${{ secrets.VAULT_ROLE_ID }}
           secretId: ${{ secrets.VAULT_SECRET_ID }}
           secrets: |
-            secret/data/products/zeebe/ci/zeebe ZEEBE_GCR_SERVICEACCOUNT_JSON;
             secret/data/products/zeebe/ci/zeebe TESTBENCH_PROD_CLIENT_SECRET;
             secret/data/products/zeebe/ci/zeebe TESTBENCH_PROD_CONTACT_POINT;
-      - name: Login to GCR
-        uses: docker/login-action@v2
-        with:
-          registry: gcr.io
-          username: _json_key
-          password: ${{ steps.secrets.outputs.ZEEBE_GCR_SERVICEACCOUNT_JSON }}
-      - uses: ./.github/actions/setup-zeebe
-      - uses: ./.github/actions/build-zeebe
-        id: build-zeebe
-      - uses: ./.github/actions/build-docker
-        with:
-          repository: ${{ env.IMAGE }}
-          version: ${{ env.TAG }}
-          push: true
-          distball: ${{ steps.build-zeebe.outputs.distball }}
-        # Executes the Testbench QA run and awaits the result
-      - name: Run Testbench QA
-        run: .ci/scripts/distribution/qa-testbench.sh
+
+      - run: |
+          businessKey="${{ env.BUILD_URL }}"
+
+          echo "Waiting for result of $businessKey"
+          jobKey=""
+
+          while true; do
+            # activate jobs non-zero exits can be ignored with `<command> || true`
+            "${zbctl}" activate jobs "$businessKey" > activationresponse.txt 2>/dev/null || true
```

Comment:
@oleschoenburg  A test run is stuck in this job. I think it is because there is no zbctl. Previously it used result from the build step. :sob:

Reply:
Oh, of course :sob: 
How about just running zbctl from within a docker container?

```
docker run --rm --entrypoint zbctl camunda/zeebe:latest activate jobs ....
```

It does propagate the exit code so I think that should work fine.
The only drawback is that you need to provide the credentials as env variables to the container.

Reply:
I'm thinking of replacing this job with a slack connector. So we can remove this wait and notify jobs.

Reply:
Yeah that's better I think :rocket:

Reply:
Moved the task to slack connector. [It works](https://camunda.slack.com/archives/C013MEVQ4M9/p1675261990586529). The process is already deployed in testbench cluster. I will open another PR for the testbench repo to add the new process.